Capstone Project

lily-coder opened the pull request:
The project I have built is based on an online website for a conference.
I am using Cindy Shin's idea in Behance (`https://www.behance.net/adagio07`)

I have created the main page and about page, both  in mobile and desktop version with breakpoints of 
Mobile: up to 768px wide.
Desktop: 768px or wider.

You can find a short video description of the project below
`https://www.loom.com/share/c505a8c86dc646888798ca2b324b6012`

hemantso commented on about.html:
```diff
@@ -0,0 +1,110 @@
+                <li><a href="#">Program</a></li>
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
```
- [x] As per design there is more space between the nav items and the about link is active. Kindly check the screenshot and fix the issue.

Yours 
![image](https://user-images.githubusercontent.com/53019916/127877306-a42da575-454d-4131-8acd-4860ba2dc973.png)

Original 
![image](https://user-images.githubusercontent.com/53019916/127877415-e643f55f-8376-440a-8f4c-eced65726c04.png)

hemantso commented on about.html:
```diff
@@ -0,0 +1,110 @@
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a href="#" class="navbar-cc-link">CC Campaign</a></li>
```
- [x] The `CC Campaign` link's colour is `#ec5242` and the font size is not matching with the original design. Kindly try to fix this issue.

Your
![image](https://user-images.githubusercontent.com/53019916/127879355-14795550-7c84-44f4-9cd2-1d4a0a1281b0.png)

Original
![image](https://user-images.githubusercontent.com/53019916/127879727-ca2b9f0a-23dd-488e-86eb-188f20a3e74b.png)

hemantso commented on index.html:
```diff
@@ -0,0 +1,126 @@
+                <li><a href="#">Join</a></li>
+                <li><a href="#">Sponsor</a></li>
+                <li><a href="#">News</a></li>
+                <li><a href="#" class="navbar-cc-link">CC Campaign</a></li>
```
- [x] The `CC Campaign` link's colour is `#ec5242` and the font size is not matching with the original design. Kindly try to fix this issue.

Your
![image](https://user-images.githubusercontent.com/53019916/127879355-14795550-7c84-44f4-9cd2-1d4a0a1281b0.png)

Original
![image](https://user-images.githubusercontent.com/53019916/127879727-ca2b9f0a-23dd-488e-86eb-188f20a3e74b.png)

hemantso commented on index.html:
```diff
@@ -0,0 +1,126 @@
+        <p class="black location">@National Museum of Korea, Art Center Nabi and more</p>
+    </section>
+    <!--THE MAIN PROGRAM SECTION-->
+    <section class="second-section">
+        <h2 class="white">Main Program</h2>
+        <div class="second-section-container">
+            <div class="prog-feature">
+                <img src="./assets/icon-1.png" alt="Lecturer">
+                <h4 class="orange">Lecture</h4>
+                <p class="white">Listen to the speakers from various countries about the message of sharing and opening</p>
+            </div>
```
- [x] The article should be placed similar to the given design. Like we can add some padding and correct the font size.

Original
![image](https://user-images.githubusercontent.com/53019916/127882691-7d6bac44-2485-4053-a453-611fe6675807.png)

Yours
![image](https://user-images.githubusercontent.com/53019916/127882763-ae82bacb-397b-4a1b-8229-9e2c64473d98.png)

hemantso commented on index.html:
```diff
@@ -0,0 +1,126 @@
+        <a href="#" class=" logo-link-2 white">SEE THE WHOLE PROGRAM</a>
+    </section>
+    <!--THE FEATURED SPEAKERS SECTION-->
+    <section class="third-section">
+        <h2>Featured Speakers</h2>
+        <aside id="my-popup"></aside>
```
- [x] The font size is too small compared to the original design and the font family applied is not correct. Kindly try to fix this issue.

Yours
![image](https://user-images.githubusercontent.com/53019916/127883459-aec5445e-9a24-45eb-917b-5b2833c4c98c.png)

Original
![image](https://user-images.githubusercontent.com/53019916/127883547-439083ba-dae4-432e-af47-0e394c5eb8be.png)

hemantso commented on index.html:
```diff
@@ -0,0 +1,126 @@
+        <div class="partners">
+            <p class="grey partners-heading">Partners</p>
+            <ul class="partners-list">
+                <li class="grey">mozilla</li>
+                <li class="grey">Google</li>
+                <li class="grey">NAVER</li>
+                <li class="grey">daumkakao</li>
+                <li class="grey">airbnb</li>
```
- [x] The font style is not correct for this section. Kindly try to correct this issue.

Yours
![image](https://user-images.githubusercontent.com/53019916/127884395-200622d2-6216-4bde-83bc-1ca51b642cab.png)

Original 
![image](https://user-images.githubusercontent.com/53019916/127884490-c249dadd-b364-49ea-8a5b-6ea37da34ca5.png)

hemantso commented on index.html:
```diff
@@ -0,0 +1,126 @@
+                <i class="fa fa-life-ring fa-3x orange"></i>
+                <p>creative commons korea</p>
+            </div>
+            <div class="footer-logo-text">
+                <p>2015 Creative Commons Korea. Some Rights Reserved</p>
+                <p>This project is a personal inspiration to showcase my skills and all content therein is not actual reality.</p>
```
- [x] The font style is not correct for this footer section. As per project requirements, we have to use cocogoose font-family. Let's try to add some more padding so that it can match the original design. Kindly try to fix this issue.

Your 
![image](https://user-images.githubusercontent.com/53019916/127885082-71df6658-4a93-4344-9d9a-778d64501206.png)

Original
![image](https://user-images.githubusercontent.com/53019916/127885164-b30bc242-cebe-4a9d-ac45-3cd3681876a1.png)

hemantso commented on about.html:
```diff
@@ -0,0 +1,110 @@
+        </div>
+        </nav>
+    <!--THE MAIN PAGE SECTION-->
+    <section class="first-section">
```
- [x] The font style is not correct for this section and for other sections of the about page. Let's try to fix this issue by applying the correct `font family` so that it should match the original design.

Your
![image](https://user-images.githubusercontent.com/53019916/127885791-bc155c9e-ce6c-4236-82dd-f299b44ec807.png)

Original
![image](https://user-images.githubusercontent.com/53019916/127885886-e4345103-93a6-4dce-9520-1e467fb5e6c7.png)

hemantso commented on index.html:
```diff
@@ -0,0 +1,126 @@
+</head>
+<body>
+    <!--THE NAVIGATION BAR SECTION-->
+    <aside class="menu-bar">
+        <i class="fa fa-close fa-2x hamburger white"></i>
+        <ul class="menu-bar-ul">
+            <li class="mobile-list"><a href="about.html" class="white">About</a></li>
+            <li class="mobile-list"><a href="#" class="white">Program</a></li>
```
- [x] On the mobile menu there is some gap above the navbar. Kindly try to fix this issue.

Your
![image](https://user-images.githubusercontent.com/53019916/127886838-2be39d00-514b-41ed-afa7-23575b17f24e.png)

hemantso commented on index.html:
```diff
@@ -0,0 +1,126 @@
+        <h2>Featured Speakers</h2>
+        <aside id="my-popup"></aside>
+    </section>
+    <footer>
+        <div class="partners">
```
- [x] Try to add the footer for the smaller screens as well.  Right now there is no footer on mobile view. Kindly try to fix this issue.

hemant-soni-vst-au4 reviewed (changes requested):
Hi @lily-coder, 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

![Alt Text](https://media.giphy.com/media/q6U4fbreC4Neo/giphy.gif)
## Required Changes ♻️
_Check the comments under the review._

- [ ] As per project requirements we have to record a video presenting the features of the project you built. It is a video with a maximum length of 5 minutes. Kindly try to record the video and add the video link to your PR's description.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- To make our project repository more professional we can segregate the CSS and Js files into different folders. like the `assets` folder.
- Try to add a summary of the project in the `README` file. Like the description of the application.

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lily-coder commented:
I have implemented all changes as requested by the reviewer

vichuge commented on index.html:
```diff
@@ -0,0 +1,133 @@
+    <!--THE FEATURED SPEAKERS SECTION-->
+    <section class="third-section">
+        <h2>Featured Speakers</h2>
+        <aside id="my-popup"></aside>
```
- [x] It looks like you make a good job for text alignment, but talking about text size is still small, please, make it the text bigger this time. 🐶
![image](https://user-images.githubusercontent.com/20563252/128196122-f5039b3e-3cf8-4247-945d-f6115781992a.png)

vichuge commented on index.html:
```diff
@@ -0,0 +1,133 @@
+        <aside id="my-popup"></aside>
+    </section>
+    <footer>
+        <div class="partners">
```
- [x] Good job using a footer for mobile size. Now let's try to use the same footer for both pages (the footer should be exactly the same). 🦃 
- [x] Look at this bug on about page footer, the text looks a little overhead. Please work on it. 🌵 
![image](https://user-images.githubusercontent.com/20563252/128196793-9fa589af-dace-42d1-9f39-1e9e6ebcbc64.png)

- [OPTIONAL] It would be excellent if the background color for the desktop size footer would be a dark color. For mobile is ok on white.

vichuge reviewed (changes requested):
# Required Changes ♻️ ♻️ ⚠️ ⚠️ 🚦 🚦 

Hi dear student, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![star_wars_almost_there](https://user-images.githubusercontent.com/20563252/121369354-644f4900-c901-11eb-9844-50c85d0f206b.gif)


## Aspects to improve ♻️
_Check the comments under the review._

- [x] Check the comments under the review. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lily-coder commented:
I have implemented the changes as requested by the reviewer as follows:

- I have increased the font size for the featured speakers section
- I have implemented the footer for both pages to be same thus fixing the bug on the mobile footer.

mricanho reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @lily-coder  🔥 

You worked hard in all the instructions that the previous Code Reviewer gives you, also a great job with your capstone!👍🏽 

![nice](https://user-images.githubusercontent.com/60631456/128201279-b1c94fd5-4c2d-4848-89f3-6aafed643a67.gif)

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
